crates/matrix-sdk/src/authentication/oidc/tests.rs

```diff
@@ -452,3 +456,48 @@ async fn test_insecure_clients() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[async_test]
+async fn test_management_url_cache() {
+    let server = MatrixMockServer::new().await;
```

Comment:
Think you don't need the mock server at all, since mocking OIDC endpoints is so adhoc (my fault!), and can use `MockClientBuilder` to create a "serverless" client.

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/4663/commits/768ec36376896384407fcfa42d968d5344402075